osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/resources/application-smr52.properties

```diff
@@ -24,3 +24,9 @@ ltediagnostic.capturetime.second=0
 ltediagnostic.capturetime.hundredths=0
 ltediagnostic.capturetime.deviation=-60
 ltediagnostic.capturetime.clockstatus=0
+
+thd.configuration.minDuration.normaltoover=0
+thd.configuration.minDuration.overtonormal=0
+thd.configuration.time.threshold=0
+thd.configuration.value.hysteresis=0
+thd.configuration.value.threshold=0
```

Comment:
Can we add some realistic values? The specification gives an example of the min duration normal to over of 15 minutes.